### ISSUE TITLE
Blade Component attribute indents on save

### ISSUE THREAD

**Issue by avr** [shufo/blade-formatter#541]
## Context (Environment)

Identical to https://github.com/shufo/blade-formatter/issues/333 - now with Laravel Component attributes.

## Current Behavior

```
<x-forms.radios
    legend="Meeting Schedule"
    name="meeting_type"
    value="single"
    v-model="form.meeting_type"
    :inline="true"
    :options="[
        'single' => ['label' => 'Default'],
        'series' => ['label' => 'Recurring meeting'],
        'scheduler' => ['label' => 'Find a meeting date'],
]" />
```

Will indent to:

```
<x-forms.radios
    legend="Meeting Schedule"
    name="meeting_type"
    value="single"
    v-model="form.meeting_type"
    :inline="true"
    :options="[
                'single' => ['label' => 'Default'],
                'series' => ['label' => 'Recurring meeting'],
                'scheduler' => ['label' => 'Find a meeting date'],
        ]" />
```

## Expected Behavior

Don't indent on save

## Possible Solution

Port fix for https://github.com/shufo/blade-formatter/issues/333

## Steps to Reproduce

https://user-images.githubusercontent.com/347482/160688772-4be14887-9525-4b64-8854-1580acd66cf8.mov

**Comment by shufo** [comment]
@avr 
Thank you. I'll take a look at it.

**Comment by shufo** [comment]
Fixed at https://github.com/shufo/blade-formatter/releases/tag/v1.22.3.

https://user-images.githubusercontent.com/1641039/161378254-4a3cfef2-376e-41ef-b915-0ee72beb97bd.mp4

I noticed after the release that the same issue seems to happen when PHP syntax error exists in the expression of attribute. I'll handle this behaviour as a syntax error in the next release.

https://user-images.githubusercontent.com/1641039/161378604-56ea9601-4854-495d-92ca-1bdf73c7aac5.mp4
